CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # TigerBeetle Changelog
 
+## 2023-11-27
+
+### Internals
+
+- [#1306](https://github.com/tigerbeetle/tigerbeetle/pull/1306),
+  [#1308](https://github.com/tigerbeetle/tigerbeetle/pull/1308)
+
+  When validating our releases, use the `release` branch instead of `main` to ensure everything is
+  in sync, and give the Java validation some retry logic to allow for delays in publishing to
+  Central.
+
+- [#1310](https://github.com/tigerbeetle/tigerbeetle/pull/1310)
+
+  Pad storage checksums from 128-bit to 256-bit. These are currently unused, but we're reserving
+  the space for AEAD tags in future.
+
+- [#1312](https://github.com/tigerbeetle/tigerbeetle/pull/1312)
+
+  Remove a trailing comma in our Java client sample code.
+
+- [#1313](https://github.com/tigerbeetle/tigerbeetle/pull/1313)
+
+  Switch `bootstrap.sh` to use spaces only for indentation and ensure it's checked by our
+  shellcheck lint.
```

Comment:
No need to change anything, but, just for overall alignment, I feel like this is probably not worth it for the changelog and is not too interesting for the reader (though, I feel like it has some explored potential for tabs vs spaces joke).